.github/cspell.json

```diff
@@ -3,43 +3,186 @@
   "language": "en",
   "flagWords": [],
   "words": [
+    "AHSL",
+    "ARGB",
     "ASDW",
+    "Akida",
```

Comment:
Names of people shouldn't be added as words into the dictionary, instead we should add `@` prefix to them in the source.